src/webgpu/shader/validation/expression/call/builtin/subgroupBroadcast.spec.ts

```diff
@@ -0,0 +1,151 @@
+export const description = `
```

Comment:
TODO: validate the builtins are not const-expression, and not an override-expression.

Reply:
Added an unimplemented test.